### DISCUSSION TITLE
How to use OrchardCore.Media.Azure with older Storage Account Version?

### DISCUSSION THREAD

**Discussion by Lanz99** [OrchardCMS/OrchardCore#10624] [Q&A]
Hello, 

is it somehow possible to use OrchardCore 1.x with older versions of storage accounts?

I am using OrchardCore version "1.0.0-rc2-13450" and would like to upgrade to the new 1.0.
The platform I am using is AzureStack and not Azure Public. For AzureStack, the latest available version of the storage account is "2019-07-07". When upgrading to OrchardCore version "1.0", the "x-ms-version" header for the request to the storage account is set to "2020-02-10", which is not known for the storage account with version "2019-07-07" and an error occurs: "x-ms-version - header has the wrong format". 

But here's where it gets tricky. The error says that the version "2020-10-02" has an incorrect format.... but the version should actually be "2020-02-10" as listed here: https://docs.microsoft.com/en-us/rest/api/storageservices/previous-azure-storage-service-versions
I'm not sure if this has any effect, since the version of my storage account is too old anyway. 

I only imported OrchardCore as NugetPackage to use it as a headless CMS.

**Comment by Piedone** [comment]
You can try to downgrade the Azure.Storage.Blobs package (simply referencing it from your root web project should do the trick) and see if that fixes your issue. OrchardCore.Media.Azure doesn't do anything I think that wasn't possible years ago so this should be safe to do. Also, during the version changes of this references package nothing else actually changed in the code base since RC2 so the older version should just work (e.g. v12.4.2 was the one used back in last May).
